tests/acme_diags/drivers/test_annual_cycle_zonal_mean_driver.py

```diff
@@ -0,0 +1,55 @@
+from unittest.case import TestCase
+from unittest.mock import Mock
+
+import numpy as np
+from cdms2.axis import TransientAxis
+from cdms2.tvariable import TransientVariable
+
+from acme_diags.driver.annual_cycle_zonal_mean_driver import _create_annual_cycle
+
+
+class TestCreateAnnualCycle(TestCase):
+    def test_returns_annual_cycle_for_a_dataset_variable(self):
```

Comment:
I tried to implement mocks for `cdms2` classes so that the test isn't dependent on `cdms2`. However, it made the test too complex and difficult to follow. 

It wouldn't provide much value anyways since if `cdms2` gets replaced, it would break the test with or without mocking.